pyscriptjs/src/utils.ts

```diff
@@ -75,9 +65,9 @@ export function handleFetchError(e: Error, singleFile: string) {
             singleFile +
             `</u> failed with error 404 (File not Found). Are your filename and path are correct?</p>`;
     } else {
-        errorContent = '<p>PyScript encountered an error while loading from file: ' + e.message + '</p>';
+        errorContent = `<p>PyScript encountered an error while loading from file: ${e.message} </p>`;
     }
-    showError(errorContent);
+    showWarning(errorContent);
```

Comment:
this is a good example of why the new approach is much better than the old one. I think that here we probably made a mistake and unintentionally allowed the execution to continue.

This function is called by `main.ts` when fetching `paths`:
https://github.com/pyscript/pyscript/blob/08d9c2fbf437bf193c1f6702b34203769cb4e22b/pyscriptjs/src/main.ts#L200-L208

If we cannot fetch a `.py` file, it is very likely that the program will break later. So I think it should be an error, not a warning.

Reply:
That makes sense, I was wondering what we should do here, I'll raise the exception then thanks!